src/android/cordovaSSDP.java

```diff
@@ -102,6 +102,7 @@ public void search(String service, CallbackContext callbackContext) throws IOExc
             multicast = new MulticastSocket(null);
             multicast.setReuseAddress(true);
             multicast.bind(srcAddress);
+            ephemeralPort = multicast.getLocalPort()
```

Comment:
Missing semi-colon :)

Reply:
Ah, yes. I tested it locally then copied over the changes to commit. Will fix.